src/cascadia/TerminalConnection/ConptyConnection.cpp

```diff
@@ -140,15 +133,8 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
             environment.as_map().insert_or_assign(L"WSLENV", wslEnv);
         }
 
-        std::vector<wchar_t> newEnvVars;
-        auto zeroNewEnv = wil::scope_exit([&]() noexcept {
-            ::SecureZeroMemory(newEnvVars.data(),
```

Comment:
This part seemed important, but I don't know why. The environment for a process isn't secret.

Reply:
(Same reason as the above I bet.)